app/utils/url/index.test.ts

```diff
@@ -27,3 +27,38 @@ describe('safeDecodeURIComponent', () => {
         expect(decoded).toBe('');
     });
 });
+
+describe('isParsableUrl', () => {
+    it('should return true for valid URLs', () => {
+        expect(isParsableUrl('http://example.com')).toBe(true);
+        expect(isParsableUrl('https://example.com')).toBe(true);
+        expect(isParsableUrl('https://example.com/path')).toBe(true);
+        expect(isParsableUrl('https://example.com:8080/path?query=1')).toBe(true);
+        expect(isParsableUrl('https://sub.domain.example.com')).toBe(true);
+        expect(isParsableUrl('ftp://example.com')).toBe(true);
+    });
+
+    it('should return false for invalid URLs', () => {
+        expect(isParsableUrl('example')).toBe(false);
```

Comment:
Is just 'example.com' a non-parsable URL?

Reply:
Yup, The `URL` constructor requires a protocol, without protocol it will throw an error and will be considered an invalid URL.

Reply:
If you can, add it to your test (if you haven't) so somebody like me and others understand that protocol is required even though 'example.com' looks valid from the naked eye.

Reply:
Done. Added a test for it.